calcom/cal.com
# feat: hashed link retrieval from event types v1 api

## What does this PR do?


- Fixes #15619 (GitHub issue number)
- Fixes CAL-4047 (Linear issue number)

Adds an optional `hashedLink` field in the event-types v1 APIs response for retrieving the private URL.

```json
"hashedLink": {
    "link": "cS4DSPAq28ALSHkyFov1y2"
}
```



## Mandatory Tasks (DO NOT REMOVE)

- [x] I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com/docs).

## How should this be tested?



- Start the v1 API and generate an API key.
- Send a GET request to http://localhost:3003/api/v1/event-types?apiKey={API_KEY}. The response should include the hashedLink field.
- Send a GET request to http://localhost:3003/api/v1/event-types/{eventTypeId}?apiKey={API_KEY}. The response should contain the hashLinked field.
